slides/lesson1/solution/versions.tf

```diff
@@ -1,4 +1,4 @@
 
 terraform {
   required_version = ">= 0.12"
-}
+}
```

Comment:
styling: let's add the blank line at the end of the file. This should be added automatically if you use [editorconfig](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) for VS Code